spec/services/request_notifier_spec.rb

```diff
@@ -42,6 +42,18 @@
 
     let(:emails) { described_class.email(:short_url_requested, short_url_request) }
 
+    context "exception testing" do
+      it "does not raise exception in integration" do
+        ENV["SENTRY_CURRENT_ENV"] = "integration"
+        expect { RequestNotifier.email(:short_url_requested, short_url_request) }.not_to raise_error(Notifications::Client::BadRequestError)
+      end
+
+      it "raises exception in production" do
```

Comment:
Could the `SENTRY_CURRENT_ENV` be explicitly set in this test?  I can foresee situations where it could potentially fail in some places because the environment is different.

Reply:
Good point, I missed that!